MicrosoftTeams.schema.json

```diff
@@ -110,6 +116,11 @@
                     "description": "A relative file path to a transparent PNG outline icon. The border color needs to be white. Size 20x20.",
                     "maxLength": 2048
                 },
+                "selectedOutline": {
+                    "type": "string",
+                    "description": "A relative file path to a transparent PNG filled icon. The fill color needs to be white. Size 20x20.",
```

Comment:
I think the outline icon needs to be 32x32 now-a-days. These comments are wrong and should be updated.

Reply:
agreed. my bad copied from wrong place